guardrails-scripts/yb-voyager-pg-grant-migration-permissions.sql

```diff
@@ -153,6 +153,19 @@ GRANT pg_read_all_stats to :voyager_user;
     \echo '--- Creating Replication Group ---'
     CREATE ROLE :replication_group;
 
+    -- Prompt the user to let them know that ownership of the tables will be transferred to the replication group and proceed only if they confirm
+    \echo ''
+    \echo 'This script will transfer ownership of all tables in the specified schemas to the specified replication group. The migration user and the original owner of the tables will be added to the replication group.'
```

Comment:
nit: You don't need to mention 'this script'. How about something like :
`Transferring ownership of all tables in the specified schemas ...`

Reply:
Done